sawtooth-java-wrapper-service/src/main/java/com/mycompany/blockchain/sawtooth/core/service/wallet/WalletValidator.java

```diff
@@ -24,7 +24,7 @@
 	@Override
 	public boolean validate(Wallet data) throws InvalidTransactionException {
 		if (data.getCustomerId() == null || data.getCustomerId().isEmpty())
-			throw new InvalidTransactionException("Asset[Name] cannot be empty or null.");
+			throw new InvalidTransactionException("Customer Id cannot be empty or null.");
```

Comment:
Use as Asset.getDEscriptor.getFieldname() while displaying . We go forward usign Proto.